scripts/python/qa-scripts/createVCAAggregate.py

```diff
@@ -0,0 +1,523 @@
+            ('A', 40), ('B:D', 120), ('E', 300), ('F', 30),
+            ('G', 300), ('H', 30), ('I', 300), ('J:K', 30), ('L', 300),
+            ('M:T', 30)
+        ]
+        vcaAggregatedFormats = [
+            ('A1:T1', self.utils.headingFormat),
+            ('A2:A', self.utils.counterFormat),
+            ('F2:F', self.utils.counterFormat),
+            ('H2:H', self.utils.counterFormat),
+            ('J2:J', self.utils.counterFormat),
+            ('K2:K', self.utils.counterFormat),
+            ('M2:M', self.utils.counterFormat),
+            ('N2:N', self.utils.counterFormat),
+            ('O2:O', self.utils.counterFormat),
+            ('P2:P', self.utils.counterFormat),
+            ('Q2:Q', self.utils.counterFormat),
+            ('R2:R', self.utils.counterFormat),
+            ('S2:S', self.utils.counterFormat),
+            ('E2:E', self.utils.noteFormat),
+            ('G2:G', self.utils.noteFormat),
+            ('I2:I', self.utils.noteFormat),
+            ('L2:L', self.utils.noteFormat),
+            ('A1:A1', self.utils.verticalHeadingFormat),
+            ('F1:F1', self.utils.verticalHeadingFormat),
+            ('H1:H1', self.utils.verticalHeadingFormat),
+            ('J1:J1', self.utils.verticalHeadingFormat),
+            ('K1:K1', self.utils.verticalHeadingFormat),
+            ('M1:S1', self.utils.verticalHeadingFormat),
+            ('M2:M', self.utils.greenFormat),
+            ('N2:N', self.utils.greenFormat),
+            ('Q2:Q', self.utils.greenFormat),
+            ('R2:R', self.utils.greenFormat),
+            ('O2:O', self.utils.yellowFormat),
+            ('S2:S', self.utils.yellowFormat)
+        ]
+
+        self.gspreadWrapper.createSheetFromDf(
+            spreadsheet,
+            'vCA Aggregated',
+            vcaAggregatedAssessments,
+            vcaAggregatedHeadings,
+            columnWidths=vcaAggregatedWidths,
+            formats=vcaAggregatedFormats
+        )
+
+
+        print('Aggregated Document created')
+        print('Link: {}'.format(spreadsheet.url))
+
+    def checkIntegrity(self, id, ass1, ass2):
+        if (
+            (ass1[self.opt.proposalIdCol] != ass2[self.opt.proposalIdCol]) or
+            (ass1[self.opt.q0Rating] != ass2[self.opt.q0Rating]) or
+            (ass1[self.opt.q1Rating] != ass2[self.opt.q1Rating]) or
+            (ass1[self.opt.q2Rating] != ass2[self.opt.q2Rating]) or
+            (ass1[self.opt.assessorCol] != ass2[self.opt.assessorCol])
+        ):
+            return False
+        return True
+
+    def checkIfMarked(self, row, column):
+        if (row[column].strip() != ''):
+            return 1
+        return 0
+
+    def calculateOutcome(self, row):
+        bad = ''
+        good = ''
+        excellent = ''
+        tot = row[self.opt.noVCAReviewsCol]
+        if (tot >= self.opt.minimumVCA):
+            if (row[self.opt.excellentCol] > (tot/2)):
+                excellent = 'x'
+            elif (row[self.opt.notValidCol] >= (tot/2)):
```

Comment:
Would like to check the requirements to see why excellent requires strict majority and filtered out does not, but I cannot find it quickly now

Reply:
You can find the reference here: https://docs.google.com/document/d/1vHapUV1LHM4cb2375V99oM32xJ-wj_XoN-DYcvKEk0A

Reply:
perfect, thanks!